robolectric/src/test/java/org/robolectric/shadows/ShadowActivityTest.java

```diff
@@ -561,6 +564,21 @@ public void setVolumeControlStream_setsTheSpecifiedStreamType() {
     assertThat(activity.getVolumeControlStream()).isEqualTo(AudioManager.STREAM_ALARM);
   }
 
+  @Test
+  public void winFrameShouldNotBeZero() {
```

Comment:
Could the test be: 

decorViewSizeEqualToDisplaySizeMinusToolBar() or something like that 

winFrame is an internal val name.